src/Home.js

```diff
@@ -0,0 +1,351 @@
+        {/* Calendar Nav */}
+        <nav className='calendarNav'>
+          <BackButton startDate={startDate} setStartDate={(setStartDate)}/>
+          {/* Week Label */}
+          <div className='weekLabel'>
+              {/* Days of the Week */}
+              <div className='daysOfWeek'>
+                {currentDates.map((date, index) => (
+                  <div key={index}><h5>{date.toLocaleString('default', {weekday: 'short'})}</h5></div>    
+                ))}
+              </div>
+              {/* Dates of the Week */}
+              <div className='daysOfWeek'>
+                {currentDates.map((date, index) => (
+                  <div key={index}><h5 className='dateStyle'>{date.getDate().toString().padStart(2, '0')}</h5></div>    
+                ))}
+              </div>
+          </div>
+          <ForwardsButton startDate={startDate} setStartDate={setStartDate}/>
+        </nav>
+      </div>
+  
+      
+    )
+  }
+  // display back button
+  const BackButton = ({startDate, setStartDate}) => {
+    const handlePrevWeek = () => {
+      const newStartDate = new Date(startDate);
+      newStartDate.setDate(startDate.getDate() - 7);
+      setStartDate(newStartDate);
+    }
+  
+    return (
+      <button className='calendarNavButton' onClick={handlePrevWeek}>
+        <img src='https://cdn-icons-png.flaticon.com/128/2989/2989988.png' 
+        alt='previous' 
+        style={{width: 25, 
+                height: 'auto', 
+                transform: 'scaleX(-1)', 
+                filter: 'invert(29%) sepia(46%) saturate(312%) hue-rotate(212deg) brightness(98%) contrast(89%)'}} />
+      </button>
+    )
+  }
+  // display forwards button
+  const ForwardsButton = ({startDate, setStartDate}) => {
+    const handleNextWeek = () => {
+      const newStartDate = new Date(startDate);
+      newStartDate.setDate(startDate.getDate() + 7);
+      setStartDate(newStartDate);
+    }
+  
+    return (
+      <button className='calendarNavButton' onClick={handleNextWeek}>
+        <img src='https://cdn-icons-png.flaticon.com/128/2989/2989988.png' 
+        alt='next' 
+        style={{width: 25, 
+                height: 'auto', 
+                filter: 'invert(29%) sepia(46%) saturate(312%) hue-rotate(212deg) brightness(98%) contrast(89%)'}} />
+      </button>
+    )
+  }
+  
+  // displays timestamps for calendar side display
+  const Times = () => {
+    return (
+      <section className='timeMarkings'>
+        {TIMES.map(time => (
+          <h5 className='timeStyle' key={time}>{time}</h5>
+        ))}
+      </section>
+    )
+  }
+  // stores all timestamps for calendar side display
+  const TIMES = [
+    '7 AM', '8 AM', '9 AM', '10 AM', '11 AM', '12 PM', 
+    '1 PM', '2 PM', '3 PM', '4 PM', '5 PM', '6 PM', 
+    '7 PM', '8 PM', '9 PM'
+  ]
+  
+  // the following 2 functions determines grid placement for a time
+  const getGridCol = (date) => {
+    switch (date.getDay()) {
+      case 0:
```

Comment:
Here, you use getDay and then map the day number to a specific column you defined in row-template-columns.
Keep in mind that columns are already indexed. The start of the first column is index 1, start of second is index 2, etc etc. This is still true if you use the `repeat` function.
So you can easily get from the date.getDay() value to the column index by doing 
`date.getDay() + 1`.